tests/models/test_tpu.py

```diff
@@ -242,16 +242,6 @@ def test_dataloaders_passed_to_fit(tmpdir):
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
-@pytest.mark.parametrize(
```

Comment:
Remove this test since `AcceleratorConnector.tpu_id` is removed, we don't need to test if it's set as expected. Its replacement `Trainer.device_ids` on TPU accelerator is tested in `tests/accelerators/test_tpu.py:: test_trainer_config_device_ids `

https://github.com/PyTorchLightning/pytorch-lightning/blob/ed2bcc5ab3e8eab45df17fee23c24d2ad8a20687/tests/accelerators/test_tpu.py#L328

Reply:
Thanks for this comment, @DuYicong515 - certainly helps for easier review. Is the alternate test you mentioned currently skipped? Do you think we should keep this test if we have a deprecated version of `tpu_id` until we remove the skip for the mentioned test below? (cc: @kaushikb11)

https://github.com/PyTorchLightning/pytorch-lightning/blob/523200971d96c6bb40262d0ca0f25c8b8ecea7b4/tests/accelerators/test_tpu.py#L316

Reply:
Thanks @krshrimali  for reviewing! 

The newly added tests passes [in a previous CI test](https://app.circleci.com/pipelines/github/PyTorchLightning/pytorch-lightning/54466/workflows/709f3bc4-0d55-48ef-9d64-f85be1a64936/jobs/152737),  however we have to disable it for now since it leads to timeout due to the use of `@pl_multiprocess_test`.  Enabling it would require optimizing existing tests and TPU CI process which requires more efforts and investigations, it's tracked in #12335.

I feel `tpu_id` property was already PyTorchLightning internal facing, and it was not used within the project, it's safe to just remove the relevant tests here. Feel free to share your thoughts here.

Reply:
Thanks for the context, @DuYicong515 - the reasoning you shared here, and in [another comment](https://github.com/PyTorchLightning/pytorch-lightning/pull/12387#discussion_r830756792) makes sense to me. Let's wait for the code-owners approval, but it sounds good to me. :)

Reply:
Just found this test which is using `tpu_id`: https://github.com/PyTorchLightning/pytorch-lightning/blob/523200971d96c6bb40262d0ca0f25c8b8ecea7b4/tests/models/test_tpu.py#L308-L316

Should this be updated as well?

Reply:
Yes, updating it now. Just noticing the TPU test failures as well. Thanks for the catch!

Reply:
Awesome, thanks!